hack/tools/requirements.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o nounset
+
+HELM_VERSION=v3.8.2
+ISTIO_VERSION=1.8.0
```

Comment:
we don't need istio

Reply:
I copied the file as is from fybrik. I'll remove reference fot unneeded tools.